### PR TITLE
Add writer set ImageIO

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

**Review by dave3d — APPROVED** [review]
Looks fine.

File: Code/IO/include/sitkImageFileWriter.h [hunk 1]
```diff
@@ -82,6 +82,23 @@ class SmartPointer;
       SITK_RETURN_SELF_TYPE_HEADER UseCompressionOff( void ) { return this->SetUseCompression(false); }
       /** @} */
 
+      /** \brief Set/Get name of ImageIO to use
+       *
+       * An option to override the automatically detected ImageIO used
+       * to write the image. The available ImageIOs are listed by the
+       * GetRegisteredImageIOs method. If the ImageIO can not be
+       * constructed an exception will be generated. If the ImageIO
+       * can not write the file an exception will be generated.
```
Comment (zivy) [inline Code/IO/include/sitkImageFileWriter.h]: The sentence "If the ImageIO can not write the file an exception will be generated." appears to be out of context here.

File: Code/IO/include/sitkImageSeriesWriter.h [hunk 2]
```diff
@@ -59,6 +63,23 @@ namespace itk {
        */
       virtual std::vector<std::string> GetRegisteredImageIOs() const;
 
+      /** \brief Set/Get name of ImageIO to use
+       *
+       * An option to override the automatically detected ImageIO used
+       * to write the image. The available ImageIOs are listed by the
+       * GetRegisteredImageIOs method. If the ImageIO can not be
+       * constructed an exception will be generated. If the ImageIO
```
Comment (zivy) [inline Code/IO/include/sitkImageSeriesWriter.h]: Same comment as given above for sitkImageFileWriter.h "If the ImageIO can not write..."

**Review by zivy — COMMENTED** [review]
Some minor documentation questions. Also, should the procedural interface ReadImage, WriteImage be updated to allow the user to specify the IO type? Not sure if it should, it already provides less control than the reader/writer classes.

**Pull request merged**